analyzer/src/funTest/assets/projects/synthetic/dotnet-expected-output.yml

```diff
@@ -233,59 +68,19 @@ packages:
 - id: "NuGet::Newtonsoft.Json:5.0.4"
   purl: "pkg:nuget/Newtonsoft.Json@5.0.4"
   authors:
-  - "James Newton-King"
+  - "James Newton-King <>"
   declared_licenses:
-  - "http://json.codeplex.com/license"
+  - "LicenseUrl: http://json.codeplex.com/license"
   declared_licenses_processed:
-    spdx_expression: "MIT"
-    mapped:
```

Comment:
@TG1999 do you know where the mapping occurs? The set of license data return now is much more complex than before (URL, name and/or expression) so I am not sure the previous mapping(s) make sense.

Reply:
Thanks for fixing this!